src/classes/ADDR_Account_TDTM.cls

```diff
@@ -36,6 +36,9 @@
 */
 public class ADDR_Account_TDTM extends TDTM_Runnable {
```

Comment:
* Classes must use `with sharing` to enforce record level access and avoid bypassing record level security
* Classes and triggers must use the `UpperCamelCase` naming convention.